### ISSUE TITLE
[New Sprite from Selection] ought to consider Timeline selections as well.

### ISSUE THREAD

**Issue by oceanhahn** [aseprite/aseprite#3623]
Presently, it is possible to transfer a Canvas selection directly to a new derivative Workspace document using [New Sprite from Selection], but the function only considers a single Cel, even if multiple Cels, or whole Layers or Frames, are selected.

It would be great if the function could be expanded to consider what is selected on the Timeline in addition to any active pixel selections in the Canvas Workspace.

Naturally, the function continues to behave as it does presently if only one Cel is active (presumably, the current view).
Conversely, a new sprite would also be generated if multiple Cels are selected *even without* a pixel selection area on the Canvas.

**In short:**
Selections on the Timeline should be applicable to New Sprite from Selection, as well as (either without or in addition to) marquee selections on the Canvas.

Thanks for considering it! :D  

..using ASEprite v1.3-beta20-x64 on Win10.

**Comment by Gasparoken** [comment]
Thanks @oceanhahn for your comments. I guess this issue is duplicated (https://github.com/aseprite/aseprite/issues/1881 which asks for this and also copying other properties).
